src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/OperatorOverloadsHaveNamedAlternates.Fixer.cs

```diff
@@ -114,12 +123,17 @@ private static async Task<Document> Fix(CodeFixContext context, SyntaxNode root,
             }
         }
 
-        private static ExpectedMethodSignature GetExpectedMethodSignature(IMethodSymbol operatorOverloadSymbol, Compilation compilation)
+        private static ExpectedMethodSignature? GetExpectedMethodSignature(IMethodSymbol operatorOverloadSymbol, Compilation compilation)
         {
             var containingType = (ITypeSymbol)operatorOverloadSymbol.ContainingType;
             ITypeSymbol returnType = operatorOverloadSymbol.ReturnType;
-            ITypeSymbol parameterType = operatorOverloadSymbol.Parameters.FirstOrDefault()?.Type;
-            string expectedName = OperatorOverloadsHaveNamedAlternatesAnalyzer.GetExpectedAlternateMethodGroup(operatorOverloadSymbol.Name, returnType, parameterType).AlternateMethod1;
+            ITypeSymbol? parameterType = operatorOverloadSymbol.Parameters.FirstOrDefault()?.Type;
+            string? expectedName = OperatorOverloadsHaveNamedAlternatesAnalyzer.GetExpectedAlternateMethodGroup(operatorOverloadSymbol.Name, returnType, parameterType)?.AlternateMethod1;
```

Comment:
Hmm I wasn't sure if there is still some code path where it can still reach here without check on the analyzer side. I'll keep the defensive code around...